apps/credential-from-input-descriptor/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:16.10.0-alpine as base
+
+WORKDIR /app
+
+FROM base as builder
+
+RUN apk add g++ make py3-pip
+
+COPY common common
+COPY rush.json .
+COPY apps/vc-api/package.json apps/vc-api/package.json
+COPY libraries/did/package.json libraries/did/package.json
+COPY libraries/webkms/package.json libraries/webkms/package.json
```

Comment:
Why are these `package.json` files needed? I would have thought that they are not needed because they are not dependencies of `credential-from-input-descriptor` as far as I can tell